src/AutoPas.h

```diff
@@ -123,7 +123,8 @@ class AutoPas {
    * @param f Functor that describes the pair-potential
    * @param dataLayoutOption useSoA Bool to decide if SoA or AoS should be used.
    */
-  void iteratePairwise(autopas::Functor<Particle, ParticleCell> *f, autopas::DataLayoutOption dataLayoutOption) {
+  template <class Functor>
+  void iteratePairwise(Functor *f, autopas::DataLayoutOption dataLayoutOption) {
```

Comment:
AutoPas::iteratePairwise should always be called with a template parameter for the function to allow inlining